aws/resource_aws_sagemaker_image_version.go

```diff
@@ -0,0 +1,124 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/sagemaker"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/service/sagemaker/finder"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/service/sagemaker/waiter"
+)
+
+func resourceAwsSagemakerImageVersion() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsSagemakerImageVersionCreate,
+		Read:   resourceAwsSagemakerImageVersionRead,
+		Update: resourceAwsSagemakerImageVersionCreate,
+		Delete: resourceAwsSagemakerImageVersionDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"image_name": {
```

Comment:
Better to have the schema keys in alphabetical order to reduce changes for https://github.com/hashicorp/terraform-provider-aws/issues/16763.